gibbon-compiler/tests/test-gibbon-examples.yaml

```diff
@@ -895,21 +895,24 @@ tests:
     failing: []
     run-modes: ["gibbon2", "gibbon3", "pointer"]
     
-  - name: layout1ContentSearchRunPipeline.hs 
+  - name: layout1ContentSearchRunPipeline.hs
+    skip: true 
     test-flags: ["--no-gc", "--opt-layout-local", "--opt-layout-use-solver"]
     dir: examples/layout_bench
     answer-file: examples/layout_bench/layout1ContentSearchRunPipeline.ans
     failing: [interp1,pointer,gibbon1, gibbon3]
     run-modes: ["gibbon2"]
     
   - name: manyFuncs.hs 
+    skip: true
```

Comment:
Mm, when will it be unskipped? Same question for the other tests you add. I hope this big of a patch will have some tests along with it...